crypto/evp/asymcipher.c

```diff
@@ -378,16 +378,16 @@ static void *evp_asym_cipher_from_dispatch(int name_id,
 
 void EVP_ASYM_CIPHER_free(EVP_ASYM_CIPHER *cipher)
 {
-    if (cipher != NULL) {
-        int i;
-
-        CRYPTO_DOWN_REF(&cipher->refcnt, &i, cipher->lock);
-        if (i > 0)
-            return;
-        ossl_provider_free(cipher->prov);
-        CRYPTO_THREAD_lock_free(cipher->lock);
-        OPENSSL_free(cipher);
-    }
+    int i;
+
+    if (cipher == NULL)
+        return;
+    CRYPTO_DOWN_REF(&cipher->refcnt, &i, cipher->lock);
+    if (i > 0)
+        return;
+    ossl_provider_free(cipher->prov);
+    CRYPTO_THREAD_lock_free(cipher->lock);
+    OPENSSL_free(cipher);
```

Comment:
This is a matter of taste, of course...  personally, I dislike this style change, and would much rather see those employing this style (like EVP_MD_free()) be changed

Reply:
>  I dislike this style change, and would much rather see those employing this style (like EVP_MD_free()) be changed

If you do that, then close this of course.  Personally I find the crawling over style bad.